specs/core/1_shard-data-chains.md

```diff
@@ -147,15 +168,24 @@ def get_shuffled_committee(state: BeaconState,
 ```python
 def get_persistent_committee(state: BeaconState,
                              shard: Shard,
-                             epoch: Epoch) -> List[ValidatorIndex]:
+                             slot: Slot) -> List[ValidatorIndex]:
```

Comment:
Should we add an assertion for the domain range of the given`slot`?

Reply:
Technically we don't need to because `generate_seed` and specifically `get_randao_mix` and `get_active_index_root` inside there are doing the check. I suppose we could add a comment saying how far back it can check.